.github/workflows/static-check.yml

```diff
@@ -26,9 +26,9 @@ jobs:
       - id: install_go
         uses: WillAbides/setup-go-faster@v1.14.0
         with:
-          go-version: "1.22.x"
+          go-version-file: "go.mod"
       - uses: dominikh/staticcheck-action@v1.4.0
         with:
           install-go: false
-          version: "2023.1.7"
```

Comment:
@tigrannajaryan, this version of staticcheck didn't support v1.23+, I've updated it to the latest released version and now all the checks execute.

Reply:
Thanks. That's what I was missing.